ROMFS/px4fmu_common/init.d/airframes/4016_holybro_px4vision

```diff
@@ -0,0 +1,140 @@
+#!/bin/sh
+#
+# @name PX4 Vision DevKit Platform
+#
+# @type Quadrotor x
+# @class Copter
+#
+# @output MAIN1 motor 1
+# @output MAIN2 motor 2
+# @output MAIN3 motor 3
+# @output MAIN4 motor 4
+#
+# @board px4_fmu-v2 exclude
+#
+
+sh /etc/init.d/rc.mc_defaults
+
+set MIXER quad_x
+set PWM_OUT 1234
+
+if [ $AUTOCNF = yes ]
+then
+	# System parameters
+	# use FMU motor outputs for less delay in the rate control loop
+	param set SYS_USE_IO 0
+
+	# Commander Parameters
+	param set COM_OBS_AVOID 1
+	param set COM_DISARM_LAND 0.5
+
+	# EKF2 parameters
+	param set EKF2_AID_MASK 35
+	param set EKF2_IMU_POS_X 0.02
+	param set EKF2_GPS_POS_X 0.055
+	param set EKF2_GPS_POS_Z -0.15
+	param set EKF2_HGT_MODE 0
+	param set EKF2_MIN_RNG 0.03
+	param set EKF2_OF_POS_X 0.055
+	param set EKF2_OF_POS_Y 0.02
+	param set EKF2_OF_POS_Z 0.065
+	param set EKF2_REQ_HDRIFT 0.3
+	param set EKF2_REQ_SACC 1
+	param set EKF2_REQ_VDRIFT 0.3
+	param set EKF2_RNG_AID 1
+	param set EKF2_RNG_A_HMAX 8
+	param set EKF2_RNG_A_VMAX 2
+	param set EKF2_RNG_POS_X 0.055
+	param set EKF2_RNG_POS_Y -0.01
+	param set EKF2_RNG_POS_Z 0.065
+	param set EKF2_PCOEF_XP -0.25
+	param set EKF2_PCOEF_YN -0.55
+	param set EKF2_PCOEF_YP -0.55
+
+	# MAVLink parameters
+	param set MAV_0_CONFIG 101
+	param set MAV_1_CONFIG 102
+	param set MAV_1_RATE 80000
+	param set MAV_1_MODE 9
+	param set SER_TEL1_BAUD 921600
+
+	# Vehicle attitude PID tuning
+	param set MC_ACRO_EXPO 0
+	param set MC_ACRO_EXPO_Y 0
+	param set MC_ACRO_P_MAX 200
+	param set MC_ACRO_R_MAX 200
+	param set MC_ACRO_SUPEXPO 0
+	param set MC_ACRO_SUPEXPOY 0
+	param set MC_ACRO_Y_MAX 150
+	param set MC_DTERM_CUTOFF 30
+	param set MC_PITCHRATE_D 0.0015
+	param set MC_PITCHRATE_I 0.2
+	param set MC_PITCHRATE_K 1
+	param set MC_PITCHRATE_P 0.15
+	param set MC_ROLLRATE_D 0.0015
+	param set MC_ROLLRATE_I 0.2
+	param set MC_ROLLRATE_K 1
+	param set MC_ROLLRATE_P 0.15
+	param set MC_YAWRATE_P 0.3
+
+	# Position Control Tuning
+	param set CP_DIST 6.0
+	param set MPC_ACC_DOWN_MAX 5
+	param set MPC_ACC_HOR_MAX 10
+	param set MPC_ACC_UP_MAX 4
+	param set MPC_MANTHR_MIN 0
+	param set MPC_MAN_Y_MAX 120
+	param set MPC_TILTMAX_AIR 45
+	param set MPC_THR_HOVER 0.3
+	param set MPC_THR_MIN 0.12
+	param set MPC_TKO_SPEED 1
+	param set MPC_VEL_MANUAL 5
+	param set MPC_XY_CRUISE 5
+	param set MPC_XY_VEL_MAX 5
+	param set MPC_XY_VEL_P 0.079
+	param set MPC_XY_TRAJ_P 0.3
+	param set MPC_Z_TRAJ_P 0.3
```

Comment:
This MPC_Z_TRAJ_P parameter is not exist now.
Related PR: https://github.com/PX4/Firmware/pull/13596